packages/grafana-ui/src/saga-themes/tokens/Theming/light.json

```diff
@@ -0,0 +1,67 @@
+{
+  "shadow": {
```

Comment:
Will we do the color tokens for light theme later?

Reply:
I'm starting with color tokens now. You can ignore that part for now.